VideoApi/VideoApi.DAL/Migrations/20210726140937_AddMagicLinkParticipantDiscriminator.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace VideoApi.DAL.Migrations
+{
+    public partial class AddMagicLinkParticipantDiscriminator : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<Guid>(
+                name: "ParticipantId",
+                table: "ParticipantStatus",
+                nullable: false,
```

Comment:
are there any existing record thats null in the ParticipantStatus table, would that cause any issue if there are any or we have ensured we are handing null in our code?

Reply:
just confirmed that in prod we have no record where the id is null so this can be marked as resolved